packages/gitbook-docs/jest.config.ts

```diff
@@ -0,0 +1,19 @@
+import type { JestConfigWithTsJest } from 'ts-jest';
+
+const jestConfig: JestConfigWithTsJest = {
+  roots: ['<rootDir>/src'],
+  testMatch: ['**/__tests__/**/*.+(ts|tsx)'],
+  moduleFileExtensions: ['ts', 'js', 'tsx'],
+  transform: {
+    // See the discussion https://github.com/vercel/next.js/issues/8663
+    '^.+\\.(ts|tsx)$': ['ts-jest', { tsconfig: 'tsconfig.jest.json' }],
```

Comment:
```suggestion
  testMatch: ['**/__tests__/**/*.+(ts)'],
  moduleFileExtensions: ['ts', 'js'],
  transform: {
    // See the discussion https://github.com/vercel/next.js/issues/8663
    '^.+\\.(ts)$': ['ts-jest', { tsconfig: 'tsconfig.jest.json' }],
```

Reply:
not required to change in this PR, but I'm not sure we are going to handle .tsx files in here.